cmd/osbuild-composer-dbjobqueue-tests/main_test.go

```diff
@@ -4,33 +4,74 @@ package main
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"os"
+	"os/exec"
+	"strings"
 	"testing"
 
+	"github.com/google/uuid"
 	"github.com/jackc/pgx/v4"
+	"github.com/stretchr/testify/require"
+
 	"github.com/osbuild/osbuild-composer/pkg/jobqueue"
 	"github.com/osbuild/osbuild-composer/pkg/jobqueue/dbjobqueue"
 
 	"github.com/osbuild/osbuild-composer/internal/jobqueue/jobqueuetest"
 )
 
+func migrate(migration string) error {
+	// migrate
+	gopath, err := exec.Command("go", "env", "GOPATH").Output()
+	if err != nil {
+		return err
+	}
+	tern := fmt.Sprintf("%s/bin/tern", strings.TrimSpace(string(gopath)))
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	if out, err := exec.Command(
+		tern,
+		"migrate",
+		"--conn-string",
+		jobqueuetest.TestDbURL(),
+		"-m",
+		fmt.Sprintf("%s/../../pkg/jobqueue/dbjobqueue/schemas", wd),
+		"-d",
+		migration,
+	).CombinedOutput(); err != nil {
+		fmt.Println("tern output:", string(out))
+		return err
+	}
+	return nil
+}
```

Comment:
Just for the record, it is possible to run migrations directly via Go call, the `tern` library does not advertise this as the author says the API is not yet fixed or stable, but it just [worked fine for provisioning service](https://github.com/RHEnVision/provisioning-backend/blob/main/internal/migrations/migrations.go#L113).

Advantage of this is that when running tests, there will be no such thing as `tern: command not found` error.

Reply:
True we can switch it over at some point. I'd prefer to get this in as the bug has been marked as blocker now.